src/ci/pgo.sh

```diff
@@ -23,8 +23,8 @@ fi
 # The various build artifacts used in other commands: to launch rustc builds, build the perf
 # collector, and run benchmarks to gather profiling data
 BUILD_ARTIFACTS=$BUILD_ROOT/build/$PGO_HOST
-RUSTC_STAGE_0=$BUILD_ARTIFACTS/stage0/bin/rustc
-CARGO_STAGE_0=$BUILD_ARTIFACTS/stage0/bin/cargo
+RUSTC_STAGE_0=$BUILD_ARTIFACTS/bootstrap-sysroot/bin/rustc
+CARGO_STAGE_0=$BUILD_ARTIFACTS/bootstrap-sysroot/bin/cargo
 RUSTC_STAGE_2=$BUILD_ARTIFACTS/stage2/bin/rustc
```

Comment:
We updated this one too, remember.
```suggestion
RUSTC_STAGE_2=$BUILD_ARTIFACTS/stage2-sysroot/bin/rustc
```
The last two lines of this script also need to be updated (lines 229 and 230).

Reply:
right, how did i miss this one :(

Reply:
done.

Reply:
Did you see the second half of my comment?
> The last two lines of this script also need to be updated (lines 229 and 230).

Reply:
yeah as always just saw the first comment, i need to be more proactive on this ones. Apologies

Reply:
here is teh image of the changes i have done as per the ask

<img width="1369" alt="image" src="https://user-images.githubusercontent.com/7916769/198103813-3e6f7002-3864-469b-b683-f390c9fd9c7e.png">

Reply:
committed, but didnt push.